moped-editor/src/layouts/DashboardLayout/TopBar.js

```diff
@@ -26,7 +25,21 @@ const useStyles = makeStyles(() => ({
 
 const TopBar = ({ className, onMobileNavOpen, ...rest }) => {
   const classes = useStyles();
-  const { logout } = useUser();
+  const { user, logout } = useUser();
+
+  const emailToInitials = email => {
+    try {
+      const subdomain = "austintexas.gov";
+      if (!email.endsWith(subdomain)) return null;
+      const [first, last] = email
+        .replace("azure_ad", "")
+        .replace(subdomain, "")
+        .split(".");
```

Comment:
Should we also capitalize the string to match the example in the issue?

Reply:
Ah, good call! Mine was running capitalized, but I don't know why

Reply:
Yes, capitalize plzzz! 🧑‍🏫

Reply:
@sergiogcx I take it that a user with a non-austintexas.gov email address will have a colored circle with no initials? that's cool w/ me.

Reply:
@johnclary Ah, sorry for the late reply. A user without initials would see a generic humanoid figure that is default fallback to Material UI:
https://material-ui.com/components/avatars/#fallbacks